cpp/src/table/table.cpp

```diff
@@ -45,7 +45,7 @@ table::table(std::vector<std::unique_ptr<column>>&& columns)
 
 // Copy the contents of a `table_view`
 table::table(table_view view, cudaStream_t stream,
-             rmm::mr::device_memory_resource* mr) {
+             rmm::mr::device_memory_resource* mr) : _num_rows{view.num_rows()} {
```

Comment:
Wow, how has this not caused problems before?

Reply:
Could you add a test to table tests that would have caught this?

Reply:
Not sure, even I was wondering same. I feel we might not be having tabe_view to table conversions.

Reply:
@jrhemstad I think it's because we convert a table to table_view all the time before we check or operate on it, and table_view gets constructed directly from columns and not with other table information.

Reply:
I will update the test case to check for num_rows and num_columns on table than converting them.